### ISSUE TITLE
Error updating dirty state editor

### ISSUE THREAD

**Issue by thSoft** [thSoft/elysium#96]
After updating to Elysium 0.5.1 and Xtext 2.8.3, when editing [my latest composition](https://github.com/thSoft/Free-Sheet-Music/tree/master/Harmath%20De%CC%81nes/112.%20zsolta%CC%81r), I get a modal error dialog saying `Error updating dirty state editor` from `org.eclipse.xtext.ui.` on every build, which is kind of showstopper. I'm not sure whether this is a bug in Elysium or Xtext, this didn't occur with Elysium 0.5.0 and Xtext 2.7.2. Stack trace:
```
org.eclipse.emf.common.util.WrappedException: java.lang.IllegalArgumentException: Illegal character in path at index 1: 1 Beatus vir.ily
	at org.eclipse.xtext.linking.lazy.LazyLinkingResource.getEObject(LazyLinkingResource.java:233)
	at org.eclipse.xtext.linking.lazy.LazyLinkingResource.doResolveLazyCrossReference(LazyLinkingResource.java:192)
	at org.eclipse.xtext.linking.lazy.LazyLinkingResource.resolveLazyCrossReference(LazyLinkingResource.java:151)
	at org.eclipse.xtext.linking.lazy.LazyLinkingResource.resolveLazyCrossReferences(LazyLinkingResource.java:137)
	at org.eclipse.xtext.EcoreUtil2.resolveLazyCrossReferences(EcoreUtil2.java:528)
	at org.eclipse.xtext.resource.impl.DefaultResourceDescription.computeReferenceDescriptions(DefaultResourceDescription.java:135)
	at org.eclipse.xtext.resource.impl.DefaultResourceDescription$2.get(DefaultResourceDescription.java:123)
	at org.eclipse.xtext.resource.impl.DefaultResourceDescription$2.get(DefaultResourceDescription.java:1)
	at org.eclipse.xtext.util.OnChangeEvictingCache.get(OnChangeEvictingCache.java:77)
	at org.eclipse.xtext.resource.impl.DefaultResourceDescription.getReferenceDescriptions(DefaultResourceDescription.java:120)
	at org.eclipse.xtext.resource.DescriptionUtils.collectOutgoingReferences(DescriptionUtils.java:29)
	at org.eclipse.xtext.resource.impl.DefaultResourceDescriptionManager.isAffected(DefaultResourceDescriptionManager.java:117)
	at org.eclipse.xtext.ui.editor.DirtyStateEditorSupport.isReparseRequired(DirtyStateEditorSupport.java:625)
	at org.eclipse.xtext.ui.editor.DirtyStateEditorSupport$UpdateEditorStateJob$1.exec(DirtyStateEditorSupport.java:157)
	at org.eclipse.xtext.ui.editor.DirtyStateEditorSupport$UpdateEditorStateJob$1.exec(DirtyStateEditorSupport.java:1)
	at org.eclipse.xtext.resource.OutdatedStateManager.exec(OutdatedStateManager.java:121)
	at org.eclipse.xtext.ui.editor.model.XtextDocument$XtextDocumentLocker.internalReadOnly(XtextDocument.java:520)
	at org.eclipse.xtext.ui.editor.model.XtextDocument$XtextDocumentLocker.readOnly(XtextDocument.java:492)
	at org.eclipse.xtext.ui.editor.model.XtextDocument.readOnly(XtextDocument.java:133)
	at org.eclipse.xtext.ui.editor.DirtyStateEditorSupport$UpdateEditorStateJob.run(DirtyStateEditorSupport.java:142)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.lang.IllegalArgumentException: Illegal character in path at index 1: 1 Beatus vir.ily
	at java.net.URI.create(URI.java:852)
	at java.net.URI.resolve(URI.java:1036)
	at org.elysium.importuri.LilyPondImportUriResolver.resolve(LilyPondImportUriResolver.java:49)
	at org.eclipse.xtext.scoping.impl.ImportUriResolver.apply(ImportUriResolver.java:56)
	at org.elysium.scoping.LilyPondImportUriGlobalScopeProvider.getAllImportedResources(LilyPondImportUriGlobalScopeProvider.java:136)
	at org.elysium.scoping.LilyPondImportUriGlobalScopeProvider.getAllImportedResources(LilyPondImportUriGlobalScopeProvider.java:142)
	at org.elysium.scoping.LilyPondImportUriGlobalScopeProvider$1.get(LilyPondImportUriGlobalScopeProvider.java:98)
	at org.elysium.scoping.LilyPondImportUriGlobalScopeProvider$1.get(LilyPondImportUriGlobalScopeProvider.java:1)
	at org.eclipse.xtext.util.OnChangeEvictingCache.get(OnChangeEvictingCache.java:77)
	at org.elysium.scoping.LilyPondImportUriGlobalScopeProvider.getImportedUris(LilyPondImportUriGlobalScopeProvider.java:94)
	at org.elysium.scoping.LilyPondImportUriGlobalScopeProvider.getScope(LilyPondImportUriGlobalScopeProvider.java:77)
	at org.eclipse.xtext.scoping.impl.AbstractGlobalScopeProvider.getScope(AbstractGlobalScopeProvider.java:49)
	at org.eclipse.xtext.scoping.impl.AbstractGlobalScopeDelegatingScopeProvider.getGlobalScope(AbstractGlobalScopeDelegatingScopeProvider.java:46)
	at org.eclipse.xtext.scoping.impl.AbstractGlobalScopeDelegatingScopeProvider.getGlobalScope(AbstractGlobalScopeDelegatingScopeProvider.java:42)
	at org.eclipse.xtext.scoping.impl.SimpleLocalScopeProvider.getScope(SimpleLocalScopeProvider.java:62)
	at org.eclipse.xtext.scoping.impl.AbstractDeclarativeScopeProvider.delegateGetScope(AbstractDeclarativeScopeProvider.java:72)
	at org.eclipse.xtext.scoping.impl.AbstractDeclarativeScopeProvider.getScope(AbstractDeclarativeScopeProvider.java:104)
	at org.eclipse.xtext.linking.impl.DefaultLinkingService.getScope(DefaultLinkingService.java:59)
	at org.eclipse.xtext.linking.impl.DefaultLinkingService.getLinkedObjects(DefaultLinkingService.java:119)
	at org.eclipse.xtext.linking.lazy.LazyLinkingResource.getEObject(LazyLinkingResource.java:250)
	at org.eclipse.xtext.linking.lazy.LazyLinkingResource.getEObject(LazyLinkingResource.java:225)
	... 20 more
Caused by: java.net.URISyntaxException: Illegal character in path at index 1: 1 Beatus vir.ily
	at java.net.URI$Parser.fail(URI.java:2848)
	at java.net.URI$Parser.checkChars(URI.java:3021)
	at java.net.URI$Parser.parseHierarchical(URI.java:3105)
	at java.net.URI$Parser.parse(URI.java:3063)
	at java.net.URI.<init>(URI.java:588)
	at java.net.URI.create(URI.java:850)
	... 40 more
```

**Comment by nittka** [comment]
Could you paste the include statements?

**Comment by nittka** [comment]
Are you really super-mega sure that it worked with Elysium 0.5 and Xtext 2.7? I reverted LilyPondImportUriResolver to the version before the fix for #44 (no changes for almost 4 years before that) and get the same error. The code always contained the code [java.net.URI].resolve(importUri) where importUri is the include string. In your case that string starts with a number (and contains whitespaces). Both cause URI.create to throw a URI-Syntax-Exception.
